acceptance/helpers/machine/machine.go

```diff
@@ -52,6 +94,22 @@ func (m *Machine) SetupAndTargetNamespace(namespace string) {
 	m.TargetNamespace(namespace)
 }
 
+func (m *Machine) GetNamespace() string {
```

Comment:
This doesn't need to be public.

Reply:
Correct. Will fix.

Reply:
Done. Commit 7eb20b7e67e5b4cff825d27bb2309b3486e1cc68.